src/main/java/hw_5/MainProgram.java

```diff
@@ -0,0 +1,20 @@
+package hw_5;
+
+import hw_5.Task2.MyInt;
+
+public class MainProgram {
+    public static void main(String[] args) {
+        MyInt a1 = new MyInt(3);
+        MyInt a2 = a1;
+
+        System.out.println(a1.x);
+        System.out.println(a2.x);
+
+        a1.x = 5;
+
+        System.out.println(a1.x);
+        System.out.println(a2.x);
+
+
```

Comment:
Where is this task?)
![Screenshot_5](https://github.com/user-attachments/assets/7f3be13a-c588-46e8-b14b-bdfeff3d56b6)

Reply:
fixed

Reply:
fixed problems, code compiled